src/tests/issues.targets

```diff
@@ -4279,7 +4258,46 @@
         </ExcludeList>
     </ItemGroup>
 
-    <ItemGroup Condition=" '$(TargetArchitecture)' == 'wasm' or ('$(TargetOS)' == 'android' or '$(TargetOS)' == 'iossimulator')">
+    <ItemGroup Condition="( '$(TargetOS)' == 'iOSSimulator' or '$(TargetOS)' == 'iOS' ) and '$(TargetArchitecture)' == 'arm64'" >
```

Comment:
@mdh1418 Is this a place where test groups could be excluded? We could try enabling small tests bundles to provide some test coverage, such as `baseservices`. After that, the next step could be to move the AOT phase to Helix, instead of the build machines, so it can be done in parallel.

Reply:
@kotlarmilos Yes, I believe issues.targets is the main way that entire test files and dirs get excluded from the runtime test build. I agree that we can start out small, `baseservices` or even `tracing/eventpipe` (as that was what the original issue denoted as problematic and what this PR initially enabled to run locally).

There seems to be already a grouping of runtime test in payloadgroups https://github.com/dotnet/runtime/blob/main/src/tests/Common/testgrouping.proj
I think we might need to have an even finer grouping, or figuring out which tests take the longest to build and why.